modules/ztp-image-based-upgrade.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-concept_{context}"]
+= Image-based upgrade for {sno} cluster with the {lcao}
+
+The {lcao} provides you with an alternative way to upgrade the platform version of a managed {sno} cluster.
+The image-based upgrade is faster than the standard upgrade method and allows you to upgrade from {product-title} <4.y> to <4.y+2> and <4.y.z> to <4.y.z+n>.
```

Comment:
nit - maybe consistency with the format of <4.y**+2**> <4.y.z**+n**>. Perhaps +n is more generic

Reply:
What I'm trying to say here is that y-stream releases are limited to 2 releases (upgrade from 4.14 to 4.16 is ok, but not to 4.18) and there's no limit to how many z-stream upgrades you can do (upgrade from 4.14.10 to 4.14.30 is okay). Hence the `n`. If it needs this much explanation, it's not clear enough

Reply:
Oh, I thought +2 was just an example. Perhaps it's just me?

Reply:
No, it could be that I need to rephrase. WDYT about this?
> ...allows you to directly upgrade from {product-title} <4.y> to <4.y+2>, and <4.y.z> to <4.y.z+n>

Does that `directly upgrade` help to emphasize what we're trying to introduce?

Reply:
Yeah that helps i think